scapy/contrib/mqtt.py

```diff
@@ -1,16 +1,19 @@
-## This file is part of Scapy
-## See http://www.secdev.org/projects/scapy for more informations
-## Copyright (C) Santiago Hernandez Ramos <shramos@protonmail.com>
-## This program is published under GPLv2 license
+''' This file is part of Scapy
```

Comment:
This is not a module docstring! This is a copyright notice that should be let as a comment.